content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,76 @@
+
+A big part of the original design was to duplicate a lot of the Envoy configuration. Notably, the listeners, clusters and routes
+with a new set of endpoints they can be referred to. The original design was created with customer requirements that
+specified client originating IPv4 requests should be proxied over IPv4, and the same for an originating IPv6 to be
+satisfied over IPv6 (we will refer to this as native IP family going forward).
+
+## Community Feedback
+
+Much of the feedback provided by the community for the original RFC was to change Envoy to better support dual-stack use cases
+internally instead of supporting this within Istio. This has led us to a [new design](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing)
+where we have taken lessons learned as well as feedback and have applied them.
+
+## Current Work
+
+We have worked with the Envoy community to address numerous concerns which is a reason why dual-stack enablement has
+taken us a while to implement. We have [matched IP Family for outbound listener](https://github.com/envoyproxy/envoy/issues/16804)
+and [supported multiple addresses per listener](https://github.com/envoyproxy/envoy/issues/11184). Alex Xu has also
+been working fervently to get long outstanding issues resolve, with the ability for Envoy to have a
+[smarter way to pick endpoints for dual-stack](https://github.com/envoyproxy/envoy/issues/21640). Some of these improvements
+to Envoy, such as the ability to [enable socket options on multiple addresses](https://github.com/envoyproxy/envoy/pull/23496),
+have landed in the upcoming Istio 1.17 release (e.g. [extra source addresses on inbound clusters](https://github.com/istio/istio/pull/41618)).
+
+The Envoy API changes the team has made can be found under their site at [Listener addresses](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto.html?highlight=additional_addresses)
+and [bind config](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig).
+Making sure we can have proper support at both the downstream and upstream connection for Envoy is important for realizing
+dual-stack support.
+
+In total the team has submitted over a dozen PRs to Envoy and are working on a least a half dozen more to make Envoy adoption of
+dual stack easier for Istio.
+
+Meanwhile, on the Istio side you can track the progress in [Issue #40394](https://github.com/istio/istio/issues/40394).
+Progress has slowed down a bit lately as we continue working with Envoy on various issues, however, we are hoping to get
+this resolved before Istio 1.17 launches in February.
+
+It is our hope that for Istio 1.17 we are able to get rudimentary support for dual-stack working. Iris Ding (Intel) and
+`Li Chun (Intel)` are already working with the community for getting redirection of network traffic for ambient. We will
```

Comment:
Is there a reason we are highlighting Li and not Iris? Spelling issue?

Reply:
It was a spelling issue. Couldn't tell since I could not get the linter working correctly for me.